meta-ibm/recipes-phosphor/webui/webui-vue/changeLogo.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+model=($(busctl get-property xyz.openbmc_project.Inventory.Manager /xyz/openbmc_project/inventory/system xyz.openbmc_project.Inventory.Decorator.Asset SubModel))
+DIR=/usr/share/www/img
+if [ "${model}" = '"S0"' || "${model}" = '"D0"' ]; then
```

Comment:
This does not execute properly. Need [] around both logic statements.
Also, model check needs to look at model[1] as the busctl response is like "s S0"

Reply:
@lxwinspur Please be sure in the future to test your code.

Reply:
@geissonator @gtmills 
Since I can't get the SubModel property on our machine, I just commented out this logic while testing this issue, so sorry

Reply:
@lxwinspur : What system is this? On any Rainier, this property must always exist

Reply:
> @lxwinspur : What system is this? On any Rainier, this property must always exist

We tested it on the fp5280g2 (DuBai) system, but the SubModel properties cannot be obtained through DBus.